spec/date_time_spec.rb

```diff
@@ -1,21 +1,17 @@
 require 'spec_helper'
 
 describe DateTime do
-
-  before do
-    @date_time = DateTime.parse("2010-01-01")
-  end
+  subject { DateTime.parse("2010-01-01") }
```

Comment:
Do not use `DateTime.parse` without zone. Use one of `Time.zone.parse`, `DateTime.parse.current`, `DateTime.parse.in_time_zone`, `DateTime.parse.utc`, `DateTime.parse.getlocal`, `DateTime.parse.iso8601`, `DateTime.parse.jisx0301`, `DateTime.parse.rfc3339`, `DateTime.parse.to_i`, `DateTime.parse.to_f` instead.